VideoLocker/res/layout/row_discussion_comments_comment.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="-5dp"
+    android:layout_marginLeft="@dimen/discussion_responses_standard_margin"
+    android:layout_marginRight="@dimen/discussion_responses_standard_margin"
+    app:cardCornerRadius="@dimen/edx_box_radius"
+    app:cardPreventCornerOverlap="false"
+    app:cardUseCompatPadding="true">
+
+    <LinearLayout
+        android:id="@+id/row_discussion_comment_layout"
+        style="@style/discussion_responses_nested_card_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:paddingTop="@dimen/discussion_responses_box_padding">
+
+        <include layout="@layout/discussion_user_profile_row" />
+
+        <TextView
+            android:id="@+id/discussion_comment_body"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor="@color/edx_grayscale_neutral_dark"
+            android:textSize="@dimen/edx_x_small"
+            tools:text="This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment." />
+
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+            <Space
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:layout_weight="1" />
```

Comment:
Instead of having to use a `LinearLayout` with a weighted `Space` just to align the view to the end, you could simply just set it's `layout_gravity` to `end` instead 😉